.eslintignore

```diff
@@ -7,4 +7,4 @@ node_modules
 packages/*/*/dist
 packages/react-aria/dist
 packages/react-stately/dist
-packages/dev/storybook-builder-preview/preview.js
+packages/dev/storybook-builder-parcel/preview.js
```

Comment:
this is the actual path and it became apparent while testing this branch locally